pkg/agent/cniserver/server.go

```diff
@@ -519,6 +520,8 @@ func (s *CNIServer) CmdAdd(ctx context.Context, request *cnipb.CniCmdRequest) (*
 		klog.Errorf("Failed to configure interfaces for container %s: %v", cniConfig.ContainerId, err)
 		return s.configInterfaceFailureResponse(err), nil
 	}
+	cniVersion := cniConfig.CNIVersion
+	cniResult, _ := result.Result.GetAsVersion(cniVersion)
```

Comment:
Should we capture the error here?

Reply:
I think it is not needed as we should always use a supported CNI version.